server/neptune/workflows/internal/prrevision/workflow.go

```diff
@@ -119,19 +125,49 @@ func (r *Runner) setRevision(ctx workflow.Context, req Request, prs []github.Pul
 	return nil
 }
 
+func (r *Runner) listModifiedFilesAsync(ctx workflow.Context, req Request, prs []github.PullRequest) map[github.PullRequest]workflow.Future {
+	now := workflow.Now(ctx)
+	taskQueue := TaskQueue
+	futuresByPullNum := map[github.PullRequest]workflow.Future{}
+
+	oldPrsCount := 0
+	for _, pr := range prs {
+		// if new version of workflow and pr is not updated within ThroughputCutOffDays, change the tq
+		version := workflow.GetVersion(ctx, version.MultiQueue, workflow.DefaultVersion, 1)
```

Comment:
Why do you need a new version for this?  TQ changes are not considered non-deterministic.  

What you probably need is a new version for is adding the sort key since that changes our event log for `list_modified_files` calls.

Reply:
Ahh that makes sense, I'll move it!